### PR TITLE
Expose handler classes in top level import

### LINKED ISSUE #47: Expose handler classes in top level import
Instead of
```python
 from baldrick.github.github_api import PullRequestHandler
```
would like to have
```python
 from baldrick import PullRequestHandler
```

### PR DESCRIPTION
**Pull request by pllim**
Fix #47 

Maybe also need to update doc? (UPDATE: Done.)

### CODE REVIEW

**Comment by astrofrog** [comment]
Yes I think we should update the API docs for this too (dropping the ``github_api`` in ``doc/api.rst``)

File: baldrick/__init__.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 from baldrick.config import load, Config
 from baldrick.blueprints import github_blueprint, circleci_blueprint
+from baldrick.github.github_api import *  # noqa
```
Comment (astrofrog) [inline baldrick/__init__.py]: Actually I agree with Stuart that we should indeed import this into ``baldrick.github``, not ``baldrick``. It's not unconceivable that we could make baldrick work with other platforms in future?

Comment (Cadair) [reply baldrick/__init__.py]: I can see me rage quitting github and moving to gitlab. Or wanting to use baldrick on my personal gitea instance ;)

Comment (pllim) [reply baldrick/__init__.py]: Okay, fair enough.

Comment (pllim) [reply baldrick/__init__.py]: Would be nice if `from baldrick import SomeHandler` is backend agnostic and something you can set as a preference but that is a battle for another day.

**Comment by pllim** [comment]
Bonus: I also cleaned up `import baldrick` namespace. It was messy and my OCD was tingling. With this patch, it now looks like:

baldrick_imports

**Pull request merged**